src/wallet/coinselection.cpp

```diff
@@ -167,6 +167,24 @@ bool SelectCoinsBnB(std::vector<OutputGroup>& utxo_pool, const CAmount& actual_t
     return true;
 }
 
+bool SelectCoinsSRD(std::vector<OutputGroup>& utxo_pool, const CAmount& target_value, std::set<CInputCoin>& out_set, CAmount& value_ret)
+{
+    out_set.clear();
+    value_ret = 0;
+
+    CAmount selected_value = 0;
```

Comment:
Consider naming this `selected_eff_value`

Reply:
Done